Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -123,10 +124,19 @@ void LogManager::Log(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type, const
   if (!log->IsEnabled() || level > log->GetLevel() || !log->HasListeners())
     return;
 
+  static size_t pos;
```

Comment:
Please don't use static here. If you want it to be fixed-position truncation like this. Make it a trait inherent of the LogManager itself (ie. Make the log manager determine this and store the cut-off position as a class variable).